packages/engine/Source/Core/IonGeocodeProvider.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Underlying geocoding services that can be used via Cesium ion.
+ *
+ * @enum {string}
+ */
+const IonGeocodeProvider = {
```

Comment:
```suggestion
const IonGeocodeProviderType = {
```

This may be a bit nit-picky, but `type` may help distinguish this from from the `*GeocoderService` instance. I got a bit confused when looking through the specs which was which.